### PR TITLE
fix: sonarcloud security hotspots review

### PR DESCRIPTION
**Pull request by arturoflores03**
Fix all the security hotspots on sonar cloud
There are 2 main security issues on diferent files:
-http protocol was change to use https
-add rel="noopener" on html links to eliminate vulnerabilities

### CODE REVIEW

File: plugin/views/admin/admin-options.php [hunk 1]
```diff
@@ -80,6 +81,21 @@ function openWelcomeMessageTransbankWebpayRest() {
...
+            b = "https://embed.typeform.com/";
+        if (!gi.call(d, id)) {
+            js = ce.call(d, "script");
+            js.id = id;
+            js.src = b + "embed.js";
+            q = gt.call(d, "script")[0];
+            q.parentNode.insertBefore(js, q)
+        }
+    })()
+</script>
```
Comment (victormendoza96) [inline plugin/views/admin/admin-options.php:101]: missing new line

Comment (Matiasnickolas) [inline plugin/views/admin/admin-options.php:101]:
```suggestion
</script>

```

File: plugin/views/admin/components/credenciales-box.php [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    <a target="_blank" href="https://transbankdevelopers.cl/documentacion/como_empezar#ambiente-de-integracion" rel="noopener">Encuentra las tarjeta de prueba acá </a>
 
     <p>
         Después de seleccionar el método de pago (en una compra de prueba), llegarás a una página de un Banco de prueba. Debes ingresar estas credenciales:
         <br>
         <strong>Rut:</strong> 11.111.111-1 <br>
         <strong>Clave:</strong> 123 <br>
     </p>
-</div>
+</div>
```
Comment (Matiasnickolas) [inline plugin/views/admin/components/credenciales-box.php:12]:
```suggestion
</div>

```

File: plugin/views/admin/components/info-validacion-webpay-oneclick-box.php [hunk 3]
```diff
@@ -12,6 +12,6 @@
 
     <br><br>
 
-    Si quieres, puedes revisar <a target="_blank" href="https://transbankdevelopers.cl/documentacion/como_empezar#puesta-en-produccion">las instrucciones detalladas de cómo pasar a producción</a>
+    Si quieres, puedes revisar <a target="_blank" href="https://transbankdevelopers.cl/documentacion/como_empezar#puesta-en-produccion" rel="noopener">las instrucciones detalladas de cómo pasar a producción</a>
 
-</div>
+</div>
```
Comment (Matiasnickolas) [inline plugin/views/admin/components/info-validacion-webpay-oneclick-box.php:17]:
```suggestion
</div>

```

File: plugin/views/admin/components/info-validacion-webpay-plus-box.php [hunk 4]
```diff
@@ -12,6 +12,6 @@
 
     <br><br>
 
-    Si quieres, puedes revisar <a target="_blank" href="https://transbankdevelopers.cl/documentacion/como_empezar#puesta-en-produccion">las instrucciones detalladas de cómo pasar a producción</a>
+    Si quieres, puedes revisar <a target="_blank" href="https://transbankdevelopers.cl/documentacion/como_empezar#puesta-en-produccion" rel="noopener">las instrucciones detalladas de cómo pasar a producción</a>
 
-</div>
+</div>
```
Comment (Matiasnickolas) [inline plugin/views/admin/components/info-validacion-webpay-plus-box.php:17]:
```suggestion
</div>

```

File: plugin/views/admin/oneclick-admin-options.php [hunk 5]
```diff
@@ -107,6 +107,21 @@ function openWelcomeMessageTransbankWebpayRest() {
...
+            b = "https://embed.typeform.com/";
+        if (!gi.call(d, id)) {
+            js = ce.call(d, "script");
+            js.id = id;
+            js.src = b + "embed.js";
+            q = gt.call(d, "script")[0];
+            q.parentNode.insertBefore(js, q)
+        }
+    })()
+</script>
```
Comment (Matiasnickolas) [inline plugin/views/admin/oneclick-admin-options.php:127]:
```suggestion
</script>

```

File: plugin/views/admin/phpinfo.php [hunk 6]
```diff
@@ -8,12 +8,10 @@
...
-               href="<?php echo admin_url('admin-ajax.php'); ?>?action=show_php_info_report"
-               target="_blank">
+            <a class="button-primary" href="<?php echo admin_url('admin-ajax.php'); ?>?action=show_php_info_report" target="_blank" rel="noopener">
                 Mostrar PHP Info en Html
             </a>
             <br>
         </fieldset>
     </div>
-</div>
+</div>
```
Comment (Matiasnickolas) [inline plugin/views/admin/phpinfo.php:17]:
```suggestion
</div>

```

**Review by Matiasnickolas — CHANGES_REQUESTED** [review]
left suggestions

**Review by Matiasnickolas — APPROVED** [review]
LGTM

**Pull request merged**